src/PackageCompiler.jl

```diff
@@ -239,11 +249,19 @@ function create_sysimg_object_file(object_file::String, packages::Vector{String}
     julia_code = """
         Base.reinit_stdio()
         @eval Sys BINDIR = ccall(:jl_get_julia_bindir, Any, ())::String
+        @eval Sys STDLIB = "$(Sys.BINDIR)/../share/julia/stdlib/v$(VERSION.major).$(VERSION.minor)"
```

Comment:
This should be enough to fix it, or? I don't see why being in the sysimage or not should matter.

Reply:
Yeah, this is the main fix. The other stuff just prevents all of the new jll's being added by default when a new sysimage is created. They would've all been added, now they're only added if used

Reply:
On Windows this ends up as a syntax error:
```julia
julia> "c:\bin\julia-1.6\bin/../share/julia/stdlib/v1.6"
ERROR: syntax: invalid escape sequence
```

With `repr` we can work around that, for instance like this:
```julia
@eval Sys STDLIB = $(repr(abspath(Sys.BINDIR, "../share/julia/stdlib", string('v', VERSION.major, '.', VERSION.minor))))
```

Reply:
Ah yes, thanks